ui/src/pages/Discord/Servers/ServerTemplate/serverTemplate.scss

```diff
@@ -13,7 +13,36 @@
     padding: 10px;
   }
   &__button {
-    background-color: transparent;
+    position: relative;
+    color: rgba(255, 255, 255, 1);
+    text-decoration: none;
+    background-color: rgb(229, 113, 229);
+    font-family: 'Yanone Kaffeesatz';
+    font-weight: 700;
+    font-size: 25px;
+    display: block;
+    padding: 8px;
+    -webkit-border-radius: 8px;
+    -moz-border-radius: 8px;
+    border-radius: 8px;
+    -webkit-box-shadow: 0px 9px 0px rgb(192, 99, 195), 0px 9px 25px rgba(0, 0, 0, .7);
+    -moz-box-shadow: 0px 9px 0px rgb(192, 99, 195), 0px 9px 25px rgba(0, 0, 0, .7);
```

Comment:
instead of having a permanent shadow why don't you make it an on-hover that gives it a shadow? With the permanent shadow, the button looks ironically aggressive